DS13/code/modules/overmap/transporter.dm

```diff
@@ -128,6 +160,19 @@
 			var/turf/open/Tu = get_turf(pick(orange(1, get_turf(eyeobj))))
 			T.send(Tu)
 		playsound(loc, 'DS13/sound/effects/transporter/send.ogg', 100, 4)
+	if(locked_on.len)
+		for(var/mob/living/H in locked_on)
+			var/area/AR = get_area(H)
+			if(!AR.linked_overmap)
```

Comment:
What if `!AR`?

Reply:
It is impossible for something to not have an area.